scripts/run_dependencies

```diff
@@ -2,7 +2,7 @@ NETWORK=beebop_nw
 VOLUME=beebop-storage
 NAME_REDIS=beebop-redis
 NAME_API=beebop-py-api
-API_BRANCH=main
+API_BRANCH=bacpop-83
```

Comment:
don't forget to change this once https://github.com/bacpop/beebop_py/pull/28 is merged